action.yml

```diff
@@ -0,0 +1,15 @@
+name: 'NPM Publish'
+description: 'NPM Publish the release'
+inputs:
+  npm-token:
+    description: 'The token used for npm publishing. Publishing to npm will be skipped if omitted.'
```

Comment:
With the proposed workflow for consuming this action, maybe this should be required? If the consumer doesn't want to publish to npm, they can just not use this action.